### ISSUE TITLE
lv_font_symbol_30_basic typo?

### ISSUE THREAD

**Issue by bengchet** [lvgl/lvgl#1074]
Hi, from the code below, I saw lv_font_symbol_30_basic, which I can't find in any files in lv_fonts folder, is it a typo?

```
    /*SYMBOL 30*/
#if USE_LV_FONT_SYMBOL_30 != 0
#if USE_LV_FONT_DEJAVU_30 != 0
    lv_font_add(&lv_font_symbol_30, &lv_font_dejavu_30);
#else
    lv_font_add(&lv_font_symbol_30_basic, NULL);
#endif
#endif
```

**Comment by embeddedt** [comment]
(cc @kisvegabor)

Looks like it doesn't exist anywhere inside the tree.

**Comment by kisvegabor** [comment]
It's really typo. Fixed in d658eef
(Also fixed in `dev-6.0`)

Thank you for reporting it.
